packages/swingset-runner/demo/zoeTests/vat-zoe.js

```diff
@@ -1,9 +1,8 @@
-/* global harden */
-
+// noinspection ES6PreferShortImport
 import { makeZoe } from '@agoric/zoe';
 
-export function buildRootObject(_vatPowers, vatParameters) {
+export function buildRootObject(__vatPowers) {
```

Comment:
Double underscore here looks to me like a typo.  If `vatParameters` is no longer being used, probably should just take the arg out entirely.  One the other hand, why is `vatParameters` no longer being used?  How is the bundle name getting to Zoe?  If the bundle name is baked into Zoe, that's arguably dangerous.  If it's not using named bundles, then something got lost in the Zoe updates and needs to be fixed.

Reply:
I investigated, and the bundle name is neither baked in, nor using named bundles.  Zoe in the `zoe-release-candidate` branch uses an import of the pre-bundled contractFacet which is built by the zoe package's `yarn build` step.  Maybe we want that to be something different, (in which case I'd suggest creating an issue) but `makeZoe` takes only one argument.

In either case, it's out of scope for this PR.

Reply:
No, those changes to `makeZoe` were merged into the master branch earlier.  If your release candidate doesn't have them, something got messed up in the merge of the RC with master and absolutely needs to be fixed.

Reply:
> No, those changes to `makeZoe` were merged into the master branch earlier. If your release candidate doesn't have them, something got messed up in the merge of the RC with master and absolutely needs to be fixed.

Fixed in #1504